nerfstudio/models/instant_ngp.py

```diff
@@ -122,7 +122,7 @@ def populate_modules(self):
         )
 
         # renderers
-        background_color = "random" if self.config.randomize_background else colors.WHITE
+        background_color = "random" if self.config.randomize_background else colors.BLACK
```

Comment:
It might be better to switch `self.config.randomize_background` to `self.config.background_color` which takes the literal `["random", "white", "black"]`